config/karma.conf.js

```diff
@@ -0,0 +1,67 @@
+// Karma configuration
+// Generated on Sun Apr 26 2015 22:55:39 GMT-0400 (EDT)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      '../src/main/resources/org/cirdles/topsoil/chart/d3.js',
```

Comment:
Should this be `../src/main/resources/org/cirdles/topsoil/chart/*.js` instead? There are JavaScript dependencies other than d3. We can also specifically enumerate them if we want to avoid including charts.